Add a "-a asciidoc7compatible" to ASCIIDOC_FLAGS.  Based on upstream
commit 5210d97c3756b04f1cf5c27fde8ed342ac756a0d

diff --git a/doc/Makefile b/doc/Makefile
index b66c37c..5d8e447 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -13,7 +13,7 @@ HTML_DIR = $(DESTDIR)$(docdir)/$(PACKAGE)/html
 PDF_DIR  = $(DESTDIR)$(docdir)/$(PACKAGE)/pdf
 
 ASCIIDOC_CONF  = $(srcdir)asciidoc.conf
-ASCIIDOC_FLAGS += -f $(ASCIIDOC_CONF) -a "builddir=$(CURDIR)/"
+ASCIIDOC_FLAGS += -f $(ASCIIDOC_CONF) -a "builddir=$(CURDIR)/" -a asciidoc7compatible
 
 #############################################################################
 # Build files
