Description: Make the build reproducible
Author: Chris Lamb <lamby@debian.org>
Last-Update: 2020-05-11
Bug-Debian: https://bugs.debian.org/960313

--- elinks-0.13.1.orig/doc/tools/asciidoc/asciidoc.py
+++ elinks-0.13.1/doc/tools/asciidoc/asciidoc.py
@@ -1061,13 +1061,9 @@ class Document:
         self.safe = True        # Default safe mode.
     def init_attrs(self):
         # Set implicit attributes.
-        d = time.localtime(time.time())
+        d = time.gmtime(int(os.environ.get('SOURCE_DATE_EPOCH', time.time())))
         self.attributes['localdate'] = time.strftime('%d-%b-%Y',d)
-        s = time.strftime('%H:%M:%S',d)
-        if time.daylight:
-            self.attributes['localtime'] = s + ' ' + time.tzname[1]
-        else:
-            self.attributes['localtime'] = s + ' ' + time.tzname[0]
+        s = time.strftime('%H:%M:%S %z',d)
         self.attributes['asciidoc-version'] = VERSION
         self.attributes['backend'] = document.backend
         self.attributes['doctype'] = document.doctype
