Description: [inpfield] Cast to unsigned char * . Refs #315
Origin: upstream, commit:adecd52b3a85a7a5964723a62e83a33dad02e849
Date: Sat Jul 13 08:17:18 2024 +0200
Bug: https://github.com/rkd77/elinks/issues/315


diff --git a/src/bfu/inpfield.c b/src/bfu/inpfield.c
index 2d6ba4ca2..f473ff2b3 100644
--- a/src/bfu/inpfield.c
+++ b/src/bfu/inpfield.c
@@ -89,9 +89,9 @@ check_number(struct dialog_data *dlg_data, struct widget_data *widget_data)
 widget_handler_status_T
 check_nonempty(struct dialog_data *dlg_data, struct widget_data *widget_data)
 {
-	char *p;
+	unsigned char *p;
 
-	for (p = widget_data->cdata; *p; p++)
+	for (p = (unsigned char *)widget_data->cdata; *p; p++)
 		if (*p > ' ')
 			return EVENT_PROCESSED;
 
